tests_end_to_end/e2e_tests_utils.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import sys
+
+import demisto_client
+
+from demisto_sdk.commands.common.tools import get_demisto_version
+
+
+def git_clone_demisto_sdk(destination_folder: str, sdk_git_branch: str = 'master'):
+    '''Clone demisto-sdk from GitHub and add it to sys.path
+    '''
+    print(f'Cloning demisto-sdk to {destination_folder}')
+    import git
+    git.Repo.clone_from(url='https://github.com/demisto/demisto-sdk.git',
+                        to_path=destination_folder,
+                        multi_options=[
+                            f'-b {sdk_git_branch}',
+                            '--single-branch',
+                            '--depth 1'
+                        ]
+                        )
+
+    sys.path.insert(1, f'{destination_folder}')
```

Comment:
is this necessary now that the e2e tests are part of the SDK?

Reply:
Yes. It's run from the Content's sdk-nightly, and TestSuite is excluded when `pip instal`ling the sdk.